src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -504,23 +517,54 @@ def reconstruct_messages(
                 tags={"use_case_id": use_case_id.value},
             )
             metrics.distribution(
-                "metrics_consumer.process_message.message.size",
-                self.__message_bytes[use_case_id],
+                "metrics_consumer.process_message.message.avg_size_in_batch",
+                self.__message_bytes[use_case_id] / self.__message_count[use_case_id],
                 tags={"use_case_id": use_case_id.value},
                 unit="byte",
             )
             metrics.distribution(
```

Comment:
I may be forgetting something, but is there a reason why these have to be a distribution?

Reply:
Yes, the second average would be for when the data points are being displayed